lib/db/models/Currency.ts

```diff
@@ -16,5 +16,18 @@ export default (sequelize: Sequelize.Sequelize, DataTypes: Sequelize.DataTypes)
 
   const Currency = sequelize.define<db.CurrencyInstance, db.CurrencyAttributes>('Currency', attributes, options);
 
+  Currency.associate = (models: Sequelize.Models) => {
+    models.Currency.hasMany(models.Pair, {
+      as: 'quoteCurrencies',
+      foreignKey: 'quoteCurrency',
+      constraints: true,
```

Comment:
Oh that's odd, I thought I'd read that `constraints` was false by default. Although I think it doesn't hurt to be explicit - for example it turned out I needed to be explicit for `foreignKey` (see https://github.com/sequelize/sequelize/issues/9328) even though the defaults *should* have worked according to the docs.

I'll go and set `constraints` to false explicitly where it's not set to true, basically where the constraints would cause a problem right now are with the `Pair` model - when we remove a pair we remove the row from the database entirely which would violate the constraints for any existing orders/swaps for that pair. I was thinking maybe for a future PR we could have an `active` flag for Pair to indicate whether it is currently supported, but still leave a row in the table.